server/plugin/command.go

```diff
@@ -347,18 +356,29 @@ func (p *Plugin) handleSubscribesAdd(_ *plugin.Context, args *model.CommandArgs,
 			}
 			subOrgMsg += "\n\n" + fmt.Sprintf("Notifications are disabled for %s", excludeMsg)
 		}
+		orgMsg := fmt.Sprintf("Github subscription, \"[%s](%s)\", with events: %s  was added to this channel by %v", owner, orgLink, subscribeEvents.String(), username.Username)
```

Comment:
I think this formatting makes sense

```suggestion
		subscriptionSuccess := fmt.Sprintf("Github subscription \"[%s](%s)\" was added to this channel by @%v, with events: %s", owner, orgLink, username.Username, subscribeEvents.String())
```